sources/scss/finder.scss

```diff
@@ -0,0 +1,140 @@
+@import"constants";
+.finder{
+	width:100%;
+
+	background-image: url("https://github.com/maxim-maidan/travel-website/blob/finder/img/explore-and-travel.png?raw=true");
+	background-repeat: no-repeat;
+	background-size: auto 90%;
+	background-position:right;
+
+	display:flex;
+	flex-direction:row;
+	padding: 0 0 0 $padding-left;
+
+	.explore{
+		display:flex;	
+		flex-direction:column;
+		h1{
+			width:470px;
+			padding:69px 0 0 0;
+
+			font-family:serif;
+			font-weight:600;
+			font-size: $title;
+			letter-spacing: -0.9px;
+			line-height:1.16;
+			color:$dark-blue;
+
+			@media screen and (max-width:576px){
+				&{
+					font-size: $title-phone;
+					width:100%;
+					padding:69px 50px 0;
+				}
+			}
+		}
+		.input{
+			width:470px;
+			height:auto;
+
+			display:flex;
+			flex-direction:row;
+			justify-content:space-between;
+			flex-wrap:wrap;
+
+			h1{
+				width:100%;
+				padding:72px 0 0 2px;
+
+				font-family: 'Roboto', sans-serif;
+				font-weight:600;
+				font-size: $subtitle;
+				letter-spacing: -0.1px;
+				line-height:1.14;
+				color:$dark-blue;
+
+				@media screen and (max-width:576px){
+					&{
+						font-size: $subtitle-phone;
+						padding:72px 0 0 0;
+					}
+				}
+			}
+			&__line{
+				margin:20px 0 50px 3px;
+				width:34px;
+				background-color:$dark-blue;
+				height:3.5px;
+
+				@media screen and (max-width:576px){
+					&{
+						margin:20px 0;
+					}
+				}
+			}
+			form{
+				width:100%;
+
+				display:flex;
+				flexlex-direction:row;
+				justify-content:space-between;
+				flex-wrap:wrap;
+				
+				.item{
+					width: calc(50% - 10px);
+					height:50px;
+					padding:0 31px 0;
+					margin:0 0 33px 0;
+
+					border-radius:5px;
+
+					font-size:$select-fs;
+
+					@media screen and (max-width:576px){
+						&{
+							width: 100%;
+							padding:1px 10px;
+							margin: 10px 0 0 0;
+						}
+					}
+				}
+			}
+			@media screen and (max-width:576px){
+				&{
+					width:100%;
+					justify-content:space-between;
+					padding:0 50px;
+				}
+			}
+		}
+		.orange{
+			margin:20px 0;
+			padding:15.5px 60px;
+
+			@media screen and (max-width:576px){
+				&{
+					margin: 20px auto;
+				}
+			}
+		}
+	}
+	@media screen and (max-width:1200px){
+		&{
+			display:flex;
+			flex-direction:row;
+			flex-wrap:wrap;
+			justify-content:start;
+			
+			padding: 0 0 0 60px;
+
+			background-size: auto 80%;
+		}
+	}
+	@media screen and (max-width:992px){
+		&{
+			justify-content:space-around;
+			background-image: none;
+			padding: 0;
+		}
+	}
+}
```

Comment:
ad new line in the end

the last line always should be empty

ex: 
![image](https://user-images.githubusercontent.com/33791517/92905671-8a0bf800-f42c-11ea-8b58-9a99c9616760.png)